src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -1054,4 +1054,10 @@ ClusterResourceScheduler::GetResourceTotals() const {
   return map;
 }
 
+bool ClusterResourceScheduler::IsLocallySchedulable(
```

Comment:
could this be const qualified?

Reply:
The `string_to_int_map_` is potentially modified inside this function.